packages/office-ui-fabric-react/src/components/Slider/examples/Slider.Basic.Example.tsx

```diff
@@ -24,7 +24,17 @@ export class SliderBasicExample extends React.Component<{}, ISliderBasicExampleS
           showValue={true}
           onChange={(value: number) => console.log(value)}
         />
-        <Slider label="Disabled example" min={50} max={500} step={50} defaultValue={300} showValue={true} disabled={true} />
+        <Slider
+          label="Snapping slider example"
+          min={1}
+          max={50}
+          step={1}
```

Comment:
I'd suggest changing this step to a higher number so that when users are playing around with this example the "snapping" is more obvious - because right now dragging it doesn't feel/look much different than the other examples.